tasks/main.yml

```
@@ -1,30 +1,36 @@
 ---
 # tasks file for pca-gophish-composition
 - name: Create the /var/pca/pca-gophish-composition directory
-  file:
+  ansible.builtin.file:
+    mode: 0755
     path: /var/pca/pca-gophish-composition
     state: directory
-    mode: 0755
 
 - name: Download and untar the pca-gophish-composition tarball
-  unarchive:
-    src: >
-      https://api.github.com/repos/cisagov/pca-gophish-composition/tarball/develop
+  ansible.builtin.unarchive:
     dest: /var/pca/pca-gophish-composition
-    remote_src: yes
     extra_opts:
       - "--strip-components=1"
+    remote_src: yes
+    src: >
+      https://api.github.com/repos/cisagov/pca-gophish-composition/tarball/develop
```

I think this should be fine with our current linting configuration.

```suggestion
    src: https://api.github.com/repos/cisagov/pca-gophish-composition/tarball/develop
```

---

Oh for the love of Zeus!!!